ci/scripts/go_build.sh

```diff
@@ -21,6 +21,9 @@ set -ex
 
 source_dir=${1}/go
 
+# Install `staticcheck`
+GO111MODULE=on go install honnef.co/go/tools/cmd/staticcheck@latest
+
```

Comment:
would it make more sense to add a docker config and make a docker image for this rather than doing the go install here as part of the build/test?